test/utils.bzl

```diff
@@ -6,6 +6,8 @@ _PLATFORM_NAME = {
     "MACOS": "macosx",
     "TVOS_DEVICE": "appletvos",
     "TVOS_SIMULATOR": "appletvsimulator",
+    "XROS_DEVICE": "xros",
+    "XROS_SIMULATOR": "xrsimulator",
```

Comment:
Pretty sure some of these (most of them except those that line up with SDK name?) should be `visionos` instead. For example, the key here should match https://github.com/bazelbuild/bazel/blob/6889d08a2f0b64eb9fe9518793ce572489b6011b/src/main/java/com/google/devtools/build/lib/rules/apple/ApplePlatform.java#L40-L41

Reply:
Hmm, a new project from Xcode in 15.2.0 still shows a lot of xrOS stuff. Odd. Minimally we need the Bazel names to match, and in the code I want to refer to it as `visionOS`.

Reply:
(So in this example, `xrsimulator` stays, but `XROS_SIMULATOR` needs to be `VISIONOS_SIMULATOR`.)

Reply:
Makes sense, and thanks for the pointer, I wasn't sure where those strings were coming from. I went ahead and updated everything to visionOS where possible. If this looks good I can take a stab at updating the tests.

Reply:
Yep, looks good.